tests/transactions/tests.py

```diff
@@ -399,9 +399,11 @@ class AtomicMiscTests(TransactionTestCase):
 
     def test_wrap_callable_instance(self):
         # Regression test for #20028
+
```

Comment:
Nope, just me changing my mind about the most readable coding style, while I was there.